server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -733,6 +733,13 @@
         </ratesource>
         <cryptologo cryptocurrency="SMART" file="smart.png"/>
     </extension>
+    <extension class="com.generalbytes.batm.server.extensions.extra.aquanow.AquanowExtension" >
+        <ratesource prefix="aquanowapi" name ="Aquanow API" >
+            <param name="price" />
+            <cryptocurrency>USD is default fiat currency.</cryptocurrency>
+        </ratesource>
+        <cryptologo cryptocurrency="SMART" file="smart.png"/>
```

Comment:
Duplicated

Reply:
Removed. We don't have any cryptocurrency.